app/_kong_plugins/ai-mcp-proxy/examples/conversion-listener.yaml

```diff
@@ -47,6 +47,8 @@ config:
       required: false
       schema:
         type: string
+    annotations:
+      title: Current weather
```

Comment:
@lmilan let's mention the affect of having annotations.title (it supposed to be auto generated in schema/configuration but let's just manually mention this for now)